content/configuration/communication/_index.md

```diff
@@ -0,0 +1,108 @@
+---
+menutitle: "Communication"
+title: "Communication"
+weight: 30
+---
+
+BotKube backend reads communication settings from two sources:
+- the [comm_config.yaml](#file-syntax) file placed in the directory specified by the **CONFIG_PATH** environment variable,
+- the exported [environment variables](#environment-variables). It takes priority over the configuration specified in the file.
+
+## File syntax
+
+The communication configuration file contains:
+
+- communication mediums configuration,

```

Comment:
Maybe `platforms` is a better word? 🤔 

BTW webhook / Elasticsearch doesn't fit that nicely into both `mediums` and `platforms` I think 🤔 But I don't have any better word for that 

```suggestion
- communication platforms configuration,

```

Reply:
It was there already but I'm ok with the change 👍